docs/software/meshtasticator/interactive-sim.mdx

```diff
@@ -0,0 +1,125 @@
+---
+id: interactive-sim
+title: Interactive Simulator Usage Guide
+sidebar_label: Interactive Event Simulator
+sidebar_position: 2
+description: Interactive Simulator usage guide for simulating multiple Meshtastic instances with TCP-based communication and configurable pathloss models.
+---
+
+The Python script _`interactiveSim.py`_ uses the [Linux native application of Meshtastic](https://meshtastic.org/docs/software/linux-native) to simulate multiple instances of the device software. These instances communicate using TCP via the script, simulating the LoRa chip. The simulator forwards messages from the sender to all nodes within range, based on their simulated positions and the selected pathloss model (see [Pathloss Model](#pathloss-model)). **Note:** Packet collisions are not yet simulated.
+
+## Usage
+
+1. Clone or download the repository and navigate to the Meshtasticator folder.
+2. (Optional) Create a virtual environment.
+3. Install dependencies:
+
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+The simulator runs the Linux native application of Meshtastic firmware. You can use either [PlatformIO](https://meshtastic.org/docs/development/firmware/build) or [Docker](https://meshtastic.org/docs/software/linux-native#usage-with-docker) to run the firmware:
+
+- **Using PlatformIO:**  
```

Comment:
I think it would be nice to have these as level 3 headings instead of bullet points. 🍺